src/api/api_auth_shibuqam.nit

```diff
@@ -94,7 +99,10 @@ class ShibCallback
 			config.players.save player
 		end
 		session.player = player
-		res.redirect "/player"
+
+		# Redirect
+		var page = session.auth_next or else "/player"
```

Comment:
The next page one? Maybe, do you have another example than login/logout where the `next` information is useful?

I can see the mission one but I don't like the semantics of passing the next uri in one case and guessing it in another case.

Following the current popcorn philosophy, this should be in the AuthHandler.